Tests/Twig/IntegrationTest.php

```diff
@@ -36,7 +38,12 @@ public function testItWorksDynamically()
                 $collectedShas['style-src'][] = $shaComputer->computeForStyle($style);
             }));
 
-        $twig = new \Twig_Environment(new \Twig_Loader_Filesystem(__DIR__.'/templates'));
+        if (class_exists('Twig\Environment')) {
```

Comment:
this can be simplified. Namespaced class names exist in Twig 1.x and 2.x too (I don't remember which minor version, but Symfony requires such minor version in all maintained branches anyway as it uses namespaced classes everywhere)

Reply:
This bundle is tested against un-maintained versions of Symfony, when installing with `composer update --prefer-lowest` we get Twig version 1.24.0 where there is no class `Twig\Environment`: 

https://github.com/twigphp/Twig/blob/v1.24.0/lib/Twig/Environment.php

Maybe we should bump Twig minimal version but that will be a BC Break.

This condition is only here to avoid a direct deprecation notices:

>   1x: Using the "Twig_Environment" class is deprecated since Twig version 2.7, use "Twig\Environment" instead.
>     1x in IntegrationTest::testItWorksDynamically from Nelmio\SecurityBundle\Tests\Twig
> 
>   1x: Using the "Twig_Loader_Filesystem" class is deprecated since Twig version 2.7, use "Twig\Loader\FilesystemLoader" instead.
>     1x in IntegrationTest::testItWorksDynamically from Nelmio\SecurityBundle\Tests\Twig

Reply:
> Maybe we should bump Twig minimal version but that will be a BC Break.

I would not consider it as a BC break, given that composer will take care of preventing to install these old unmaintained Twig releases alongside the new version of this bundle.